packages/aws-cdk-lib/aws-batch/lib/ecs-container-definition.ts

```diff
@@ -1009,6 +1023,20 @@ export interface EcsFargateContainerDefinitionProps extends EcsContainerDefiniti
    * @default - 20 GiB
    */
   readonly ephemeralStorageSize?: Size;
+
+  /**
+   * The vCPU architecture of Fargate Runtime.
+   *
+   * @default - X86_64
```

Comment:
For my own understanding, why set defaults for this and `fargateOperatingSystemFamily`?

Reply:
My understanding is the default value should be same as the default value in [CFN resource](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-batch-jobdefinition-runtimeplatform.html).